### PR TITLE
[ci] jail //python/ray/serve:test_websockets

### PR DESCRIPTION
**Pull request by can-anyscale**
//python/ray/serve:test_websockets is becoming very flaky, jail it

Screenshot 2023-09-07 at 12 51 30 AM

Test:
- CI

### CODE REVIEW

**Review by edoakes — APPROVED** [review]
thx, will fix it after

**Review by aslonnie — APPROVED** [review]

**Comment by aslonnie** [comment]
> thx, will fix it after

do you have a pr/link/something that we can track the progress of the fixing?

**Comment by sihanwang41** [comment]
Hi @aslonnie and @can-anyscale  looks like the test is not being run. here is my pr: https://buildkite.com/ray-project/premerge/builds/5248#018a71c9-6550-4f3e-bd16-12bbe7132bfc

**Comment by aslonnie** [comment]
the test did run, but in 3 parallel dockers, and only one has tests.

@can-anyscale ..

File: .buildkite/serve.rayci.yml [hunk 1]
```diff
@@ -36,3 +36,11 @@ steps:
         --test-env=RAY_SERVE_ENABLE_EXPERIMENTAL_STREAMING=0
     depends_on: servebuild
     job_env: servebuild
+
+  - label: ":ray-serve: serve: flaky tests"
+    instance_type: medium
+    soft_fail: true
+    commands:
+      - bazel run //ci/ray_ci:test_in_docker -- //... serve --run-flaky-tests --parallelism-per-worker 3
```
Comment (aslonnie) [inline .buildkite/serve.rayci.yml:44]: @sihanwang41 , your PR https://github.com/ray-project/ray/pull/39413 seems to have fixed the test, so you can revert this PR's change in your PR.

@can-anyscale 
- maybe remove the `--parallelism-per-worker 3` ?
- and/or fix the issue of running 0 tests in a parallel container.

**Comment by can-anyscale** [comment]
oops my bad, I'll fix the 0 test issues

**Pull request merged**